js/main.js

```diff
@@ -1,5 +1,8 @@
 import {createPhotos} from './mockPhotos.js';
 import {renderPhotos} from './renderingPicture.js';
+import { addHandler } from './popupGallery.js';
 
 const photos = createPhotos();
 renderPhotos(photos);
+addHandler(photos);
```

Comment:
`addHandler` не совсем корректно, поскольку идея не в том, чтобы установить обработчик, он уже там установлен и работает, а в том, чтобы передать в модуль галереи данные наших фотографий.
Возможно, даже в будущем мы перенесем вызов этого модуля внутрь renderPhotos, но можно и нет, но на данный момент это вот как-то так:
```suggestion
setPhotos(photos);
```
Мы передаем внуть нашей галереи текущие "актуальные" данные, мы можем изменить фотографии и установить новый массив и галерея должна так же функционировать.